akka-stream/src/main/mima-filters/2.5.x.backwards.excludes

```diff
@@ -0,0 +1,37 @@
+# Java9 RS APIs and more that was missed in a few earlier versions
```

Comment:
Weird... ok?

Reply:
yeah, I don't fully understand how these files are used

Reply:
somewhat similar thing in master https://github.com/akka/akka/pull/26995